pkg/config/network.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import "fmt"
+
+type Networks struct {
+	Default string
+	All     map[string]*Network
+}
+
+type Network struct {
+	URLs []string
+}
+
+// SetDefault changes default network.
+func (n *Networks) SetDefault(name string) error {
```

Comment:
Since Network is meant to be a read-only, perhaps we can enforce it even further by implementing this not as a setter, but as a builder pattern. Usage would look like this:

```
  config.DefaultNetworks = config.DefaultNetworks.WithDefault("mainnet")
```

Complimentary to that would be `NewNetworks` that would take the default, and `map[string]Network`, and return initialized `Networks`.

This interface should take into consideration that in the future Networks would be fetched from the chain.

Reply:
I have implemented AddNetwork function. When blockchain returns list of brokers, we can do something in the spirit of:
```go
brokersNet := resolveBrokersNetFromChain
network.Add("chain-brokers", brokersNet)
```